lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -283,7 +284,8 @@ def add_destination(post, options)
       end
 
       def add_expand_parameters(post, options)
-        post[:expand] = Array.wrap(options[:expand])
+        post[:expand] ||= []
+        post[:expand].concat(Array.wrap(options[:expand]).map(&:to_sym)).uniq!
```

Comment:
I see what you mean, unfortunately `Array.wrap` doesn't play nicely with `Set`s, ends up building an array with the `Set` as its element. Would rather keep using this nice `Array.wrap` to handle `options[:expand]` being `nil`, `'charge'`, `['charge', 'somethig_else']`.
```
irb(main):001:0> Array.wrap(Set.new(['a']))
=> [#<Set: {"a"}>]
```